main/exe/src/euCliReader.cxx

```diff
@@ -34,6 +34,25 @@ int main(int /*argc*/, const char **argv) {
   uint32_t timestamph_v = timestamph.Value();
   bool not_all_zero = eventl_v||eventh_v||triggerl_v||triggerh_v||timestampl_v||timestamph_v;
 
+
+  // load configuration
+    // empty configuration object, prevents crash if no config file given
+  eudaq::Configuration eu_cfg( "", "" );
+  std::ifstream conffile( file_conf.Value() );
+  if( conffile ){
+    eu_cfg.Load( conffile, std::string("euCliReader") );
+  }
+  // prevent warning if no config file name given
+  else if ( file_conf.Value().empty() ){}
+  // but warn if attempt failed
+  else{ 
+    std::cout << "WARNING, config file '" << file_conf.Value() << "' not found!" << std::endl;
+  }    
+  // build shared pointer
+  const eudaq::Configuration const_eu_cfg = eu_cfg;
```

Comment:
Are you using `const_eu_cfg` ever?

Reply:
I am now using it in the line below. This is the way it is used done in the corry EUDAQ2 converter.